### PR TITLE
[CCA-28] update GPU documentation for public facing site

### PR DESCRIPTION
**Pull request by j1hun**
### What does this PR do? What is the motivation?

- updates CCM's public docs to reflect GPU container cost availability

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Clouds
 Resources
 : CCM allocates costs for Kubernetes clusters and includes cost analysis for many associated resources such as Kubernetes persistent volumes used by your pods.
 
-CCM displays costs for resources including CPU, memory, and more depending on the cloud and orchestrator you are using on the [**Containers** page][1].
+CCM displays costs for resources including CPU, memory, and GPU and more depending on the cloud and orchestrator you are using on the [**Containers** page][1].
```
Comment (elihschiff) [inline content/en/cloud_cost_management/container_cost_allocation.md:27]: This has 2 `and`s

I would vote to just not mention GPUs here and the `and more` should cover that

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 2]
```diff
@@ -193,19 +200,19 @@ Use the `allocated_spend_type` tag to visualize the spend category associated wi
...
-| Cluster idle | Cost of resources (such as memory and CPU) that are not reserved by workloads in a cluster. This is the difference between the total cost of the resources and what is allocated to workloads. |
+| Usage | Cost of resources (such as memory, CPU, and GPU) used by workloads, based on the average usage on that day. |
+| Workload idle | Cost of resources (such as memory, CPU, and GPU) that are reserved and allocated but not used by workloads. This is the difference between the total resources requested and the average usage. |
+| Cluster idle | Cost of resources (such as memory, CPU, and GPU) that are not reserved by workloads in a cluster. This is the difference between the total cost of the resources and what is allocated to workloads. |
 
 ### Persistent volume
 
-The cost of an AWS EBS volume has three components: IOPS, throughput, and storage. Each is allocated according to a pod's usage when the volume is mounted.
+The cost of an EBS volume has three components: IOPS, throughput, and storage. Each is allocated according to a pod's usage when the volume is mounted.
```
Comment (elihschiff) [inline content/en/cloud_cost_management/container_cost_allocation.md:215]: Any reason why you made this change?

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 3]
```diff
@@ -332,7 +339,7 @@ In addition to Kubernetes pod and Kubernetes node tags, the following out-of-the
 | `persistent_volume_reclaim_policy`      | The Kubernetes Reclaim Policy on the Persistent Volume.                                                                                      |
 | `storage_class_name`                    | The Kubernetes Storage Class used to instantiate the Persistent Volume.                                                                      |
 | `volume_mode`                           | The Volume Mode of the Persistent Volume.                                                                                                    |
-| `ebs_volume_type`                       | The type of the AWS EBS volume. Can be `gp3`, `gp2`, or others.                                                                              |
+| `ebs_volume_type`                       | The type of the EBS volume. Can be `gp3`, `gp2`, or others.                                                                              |
```
Comment (elihschiff) [inline content/en/cloud_cost_management/container_cost_allocation.md:342]: Any reason you removed the AWS here?

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 4]
```diff
@@ -40,18 +40,21 @@ The following table presents the list of collected features and the minimal Agen
 | Feature | Minimal Agent version | Minimal Cluster Agent version |
 |---|---|---|
 | Container Cost Allocation | 7.27.0 | 1.11.0 |
+| GPU Container Cost Allocation | 7.54.0 | 7.54.0 |
```
Comment (kaankoseler) [inline content/en/cloud_cost_management/container_cost_allocation.md:43]: When you asked the #container-app team were they unable to provide a corresponding cluster agent version? This doesn't seem right to me

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 5]
```diff
@@ -63,9 +66,11 @@ The following table presents the list of collected features and the minimal Agen
...
```
Comment (kaankoseler) [inline content/en/cloud_cost_management/container_cost_allocation.md:69]: Same comment as above, is there no corresponding cluster agent version?

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 6]
```diff
@@ -81,9 +86,11 @@ The following table presents the list of collected features and the minimal Agen
...
```
Comment (kaankoseler) [inline content/en/cloud_cost_management/container_cost_allocation.md:89]: Same comment as above

File: content/en/cloud_cost_management/container_cost_allocation.md (hunk 3 above)
Comment (j1hun) [reply content/en/cloud_cost_management/container_cost_allocation.md:342]: was failing the vale check, reason was it did not approve of having 'AWS EBS' so maybe new linter rules were added?

File: content/en/cloud_cost_management/container_cost_allocation.md (hunk 2 above)
Comment (j1hun) [reply content/en/cloud_cost_management/container_cost_allocation.md:215]: was failing the vale check, reason was it did not approve of having 'AWS EBS' so maybe new linter rules were added?

File: content/en/cloud_cost_management/container_cost_allocation.md (hunk 4 above)
Comment (j1hun) [reply content/en/cloud_cost_management/container_cost_allocation.md:43]: After 1.11 they changed to using a 7.xx version numbering convention. It didn't seem like they had a corresponding cluster agent version. I discussed with @enemyghost and we decided on using 7.54.0 for both

**Review by elihschiff — APPROVED** [review]

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 7]
```diff
@@ -222,30 +229,30 @@ The cost of an AWS EBS volume has three components: IOPS, throughput, and storag
 
 ### Compute
 
-The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
+The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. If there is GPU usage on the pod, the cost of the host instance is split into three components: 95% for the GPU, 3% for the CPU, and 2% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
```
Comment (enemyghost) [inline content/en/cloud_cost_management/container_cost_allocation.md:232]: same edit as above regarding host vs pod

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 8]
```diff
@@ -222,30 +229,30 @@ The cost of an AWS EBS volume has three components: IOPS, throughput, and storag
...
+| Workload idle | Cost of resources (such as memory, CPU, and GPU) that are reserved and allocated but not used by workloads. This is the difference between the total resources requested and the average usage. |
+| Cluster idle | Cost of resources (such as memory, CPU, and GPU) that are not reserved by workloads in a cluster. This is the difference between the total cost of the resources and what is allocated to workloads. |
 
 {{% /tab %}}
 {{% tab "Google" %}}
 
 ### Compute
 
-The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
+The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. If there is GPU usage on the pod, the cost of the host instance is split into three components: 95% for the GPU, 3% for the CPU, and 2% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
```
Comment (enemyghost) [inline content/en/cloud_cost_management/container_cost_allocation.md:247]: same edit

**Review by enemyghost — APPROVED** [review]

**Review by jhgilbert — APPROVED** [review]

**Pull request merged**